cara/apps/calculator/__init__.py

```diff
@@ -252,7 +285,7 @@ def make_app(
         template_environment=template_environment,
         default_handler_class=Missing404Handler,
         report_generator=ReportGenerator(loader, calculator_prefix),
-        xsrf_cookies=True,
+        xsrf_cookies=xsrf_cookies,
```

Comment:
See my other comment about the endpoint being exposed (via GET), I don't believe this is the right thing to be doing.

Reply:
See my comment above.